### ISSUE TITLE
nvda is not reading any information in TeamViewer

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14243]
### Steps to reproduce:
- open TeamViewer
- walk with tab

### Actual behavior:
NVDA is silent

log: [teamviwer.txt](https://github.com/nvaccess/nvda/files/9770105/teamviwer.txt)

### Expected behavior:
NVDA should be able to read the connect button, or when it returns it should be able to re-announce the edit.

the software TeamViewer seems to be having very big accessibility issues as even the narrator can't read anything in the app.
This was a regression in TeamViewer which in previous versions was much more accessible.
I open this call to nv access because using jaws the reader was able to read at least the edit box and the connect button which already allowed me to access another machine.
so I think that even if TeamViewer is the big culprit here it is still possible to give the nvda user a better condition of use.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
`nvda.exe, NVDA alpha-26777,f4955df6`

#### Windows version:
`windows 11 22623.741`

#### Name and version of other software in use when reproducing the issue:
`TeamViewer.exe, TeamViewer 15.34.4.0`

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by seanbudd** [comment]
>This was a regression in TeamViewer which in previous versions was much more accessible.

Just to confirm, this recently broke by updating TeamViewer?
Have you reported this to the TeamViewer developers?
Can this be reproduced with older versions of NVDA (e.g. 2022.2/2022.3)?
If this is a regression in TeamViewer, it is unlikely we can anything about it.

**Comment by fernando-jose-silva** [comment]
Just to confirm, has this recently broken updating TeamViewer?
yea

Did you report this to the TeamViewer developers?
no, I don't know how to do this, I have to search to find an accessibility channel for them

Can this be reproduced with older versions of NVDA (eg 2022.2/2022.3)?
yea

If this is a regression in TeamViewer it is unlikely that we can do anything about it.
I opened the case because jaws has a better performance than nvda, although far from ideal, I believe that nvda can reach at least the performance obtained by the other screen reader.

**Comment by seanbudd** [comment]
>no, I don't know how to do this, I have to search to find an accessibility channel for them

The Team Viewer website has plenty of contact channels.
https://www.teamviewer.com/en-us/customer-support/

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still seeing this issue with the last version of Teamviewer and NVDA 2024.3 Beta4?

**Comment by fernando-jose-silva** [comment]
Current test results were:
when browsing with tabe, nvda does not report anything.
When you press nvda + tabe, nvda displays panel.
If I remove the focus from the window and then return the focus mainly by opening the nvda menu and closing it, nvda announces the item with focus.
So I think that if nvda does something here we can access the application better, since the objects are accessible, only when walking with tab and for some reason nvda does not recognize the object in focus.
the narrator says nothing, even removing the focus from the window and returning.
log below.
nvda.exe, NVDA alpha-32997,3f82ab67
TeamViewer.exe, TeamViewer 15.55.3.0
windows 11 26100.150
[viwer.txt](https://github.com/user-attachments/files/16322730/viwer.txt)

**Comment by Adriani90** [comment]
I think teamviewer displays an overlay window with something else, pressing escape closes that overlay and you can navigate with tab and shift+tab in Teamviewer as expected.

**Comment by Adriani90** [comment]
Given the overlay window can be closed by pressing escape, and NVDA cannot do anything about that overlay, I am closing as can't fix. Please contact the teamviewer developers via this link:
https://login.teamviewer.com/LogOn/PopUp/SupportCenter?returnUrl=%2Foauth2%2Fauthorize%2FSupportCenter%3Fdisplay%3Dpopup%26response_type%3Dcode%26client_id%3D296342-d1oqaqFYH0fS06yQNX7U%26scope%3D%26state%3Dfwst_23a8927125662e71bf5b8adf9b3eda43b7cd775aa4a1f014e4e90a65ff9a48dfd1efa5945d410c13fdcf8959747568000184daa1d2881f650a60d9d84553e418c138670d048a39b0eea0dd6ca9848393a4de8b89b719847a227311f6173d04170338ee4283e1f3d6cb763458f73861ac%26redirect_uri%3Dhttps%253a%252f%252ffreshworks-help175.freshworks.com%252fsp%252fOAUTH%252f177803963570753262%252fcallback%26registration_id%3D177803963570753262

Or this link:
https://www.teamviewer.com/en/global/support/customer-support/
